tests/AutomationApp/Microsoft.Identity.Test.UIAutomation/UserInformationFieldIds.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Test.LabInfrastructure;
+
+namespace Microsoft.Identity.Test.UIAutomation
+{
+    public class UserInformationFieldIds
```

Comment:
We already have identical objects in Microsoft.Identity.Test.UIAutomation.Infrastructure - is this just temporary for the POC?